ExpressFirstApp/tests/todo.test.js

```diff
@@ -99,6 +99,24 @@ describe('POST /todo', () => {
 });
 
 describe('DELETE /todo', () => {
+  it('should throw an error reading the file due to mock of fs.readFile', async () => {
```

Comment:
you are not testing the mock itself
better naming will be "should throw an error while trying to delete tasks"